### ISSUE TITLE
NVDA not reading aria-live field in Google Chrome + React

### ISSUE THREAD

**Issue by vinaychaudhary1504** [nvaccess/nvda#10677]
You have arrived at a default issue template (bug report).
It is preferable to [choose the type of issue](https://github.com/nvaccess/nvda/issues/new/choose) to submit.
Hint: Swap to the preview tab to be able to click the links.
Direct links to the templates:
- [Bug report](https://github.com/nvaccess/nvda/issues/new?template=bug_report.md)
- [Feature request](https://github.com/nvaccess/nvda/issues/new?template=feature_request.md)

### Steps to reproduce:
Have a react application with aria-live="polite" and aria-atomic="true" field.

### Actual behavior:
It should read the content of aria-live field.

### Expected behavior:
It is reading it sometimes and sometime it is ignoring it.
Same Code is working in google chrome

### System configuration
Windows 10 , Google Chrome V79.0

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
2019.2

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
yes, 2017

**Comment by ehollig** [comment]
Can you provide a minimal test case that demonstrates the issue?

**Comment by jurokapsiar** [comment]
This does not seem to be valid anymore. I cannot reproduce it with NVDA 2020.1 and Chrome Version 83.0.4103.116 (Official Build) (32-bit).

https://codesandbox.io/s/laughing-elgamal-dg6mp?file=/example.js - clicking on the button causes NVDA to read the live narration correctly.

**Comment by Adriani90** [comment]
Thanks for the update. I am closing as works for me. Please comment on this if you still have the issue and we can reopen it.
